src/analyze/accumulatedtracedata.cpp

```diff
@@ -195,7 +239,17 @@ bool AccumulatedTraceData::read(istream& in, const ParsePass pass, bool isRepars
     // allocations, i.e. when a deallocation follows with the same data
     uint64_t lastAllocationPtr = 0;
 
+    auto const & filtIn = dynamic_cast<boost::iostreams::filtering_istream &>(in);
```

Comment:
please don't abbreviate variable names, this could be `filteringIn`

Reply:
Will fix.

By the way, looking at this I'm realizing the interface in theory allows someone to pass in a non-`filtering_istream` in theory, and I'm completely ignoring that with this `dynamic_cast` Is what I'm doing here acceptable to you?

I could imagine either:

 * switching the `dynamic_cast` to a pointer cast and throwing the byte counts inside an `if`-block to still allow for non-`filtering_istream` calls
* switching the function signature to only accept `filtering_istream`. 

thoughts?

Reply:
I went with changing the interface, but happy to change to the run-time solution if you prefer that.